### PR TITLE
Bug 1137766 - Error in both test_settings_change_language tests

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5bbf455ebb7f180c1d80dd12cd03f26dbe1807ae

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py [hunk 1]
```diff
@@ -10,17 +10,20 @@
...
 
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.data_layer.set_setting('devtools.pseudolocalization.enabled', True)
+
     def test_change_language_settings(self):
         lang_name = self.marionette.execute_script("""
-            var  = window.wrappedJSObject.navigator.mozL10n.;
-            return ['fr-x-psaccent'].translate('Packaged Accented');
+            var qps = window.wrappedJSObject.navigator.mozL10n.qps;
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py]: What is a qps?

Comment (stasm) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py]: Locale codes starting with "q" can be used to describe custom languages. "ps" is short for "pseudolocalization". For example, Microsoft uses "qps-ploc" as a code for one of their pseudolocalizations.

We're moving away from this scheme in favor of "fr-x-psaccent" and "ar-x-psbidi", where "-x-" is also used as a means of introducing a custom extension.  The reason for this move is that `Intl` and other language negotiation mechanisms give up completely on "qps-ploc";  however, with "ar-x-psbidi", they usually understand that we want something in Arabic and for example can properly format date and time.

Comment (stasm) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py]: In l20n.js, we removed the `mozL10n.qps` API and called it: `document.l10n.pseudo`.  The Settings app doesn't use l20n yet, however, and shared/js/l10n.js still uses "qps" as the name of the property.

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py]: I don't really know, but that was what was accidentally removed in:
https://github.com/mozilla-b2g/gaia/commit/f03380e451eb2a33a27e7b30c183c311f8e7430d#diff-9413f92730386581ed79f16a649180fa
(from Bug 1217733)

**Pull request merged**